storage/innobase/btr/btr0bulk.cc

```diff
@@ -62,6 +62,10 @@ PageBulk::init()
 		the allocation order, and we will always generate redo log
 		for page allocation, even when creating a new tablespace. */
 		mtr_start(&alloc_mtr);
+		if (dict_index_get_space(m_index)
```

Comment:
So infact is generic enhancement. As in alloc_mtr doesn't need to redo log if index to rebuild is temporary.
Can we have a comment explaining this info here.

Interestingly this could be files as upstream optimization patch too. Developer to decide.